src/components/layout/component.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import Link from 'redux-first-router-link';
+import { breakpoints } from 'utils/responsive';
+
+import logo from 'components/layout/logo.svg';
+
+// components
+import DesktopLayout from './desktop';
+import MobileLayout from './mobile';
+
+import FooterMenu from 'components/bar-menu/footer-menu/component';
+import Map from 'components/map-container';
+import styles from './style.module.scss';
+
+const Layout = ({ mapView, location }) => {
+  const isMobile = window.innerWidth < breakpoints.lg;
+
+  return (
+    <div className={cx([styles.printOnly_wrapper])}>
+      <Link to={{ type: 'PAGE/APP' }}>
+        <img className={styles.logo} src={logo} alt="mangrove-atlas" />
```

Comment:
This is ok, but I think for assets we should not to import. Because:

1. We always should save an optimized asset in our directories.
2. SVG files don't need to be optimized as an image
3. Increase the project size on the compilation (webpack)

So, they should be stored in the public directory and put the URL, no importation.